app/Services/Campaign/ExportService.php

```diff
@@ -142,27 +144,19 @@ protected function gallery(): self
         return $this;
     }
 
-    protected function processFolder($image, string $path): string
+    protected function processImage($image): self
     {
-        $route = $path . '/' . $image->name;
-
-        if ($image->imageFolder) {
-            $route = $route . '/';
-            $route = $this->processFolder($image->imageFolder, $route);
-        } else {
-            $route = $route . '.' . $image->ext;
+        if(!$this->assets) {
+            $this->archive->addFromString('gallery/' . Str::slug($image->name) . '.json', $image->export());
```

Comment:
I have multiple images called "screenshot" which is resulting in lost data. Please include the model's uuid or id (not sure if we have ids on this?) in the file name :to prevent data loss :-)